timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/AsyncTimeLockServiceMetadataTest.java

```diff
@@ -134,16 +131,11 @@ private List<Optional<LockRequestMetadata>> getAllLockEventsMetadata() {
     }
 
     private List<LockWatchEvent> getAllLockWatchEvents() {
-        ConjureStartTransactionsResponse response =
-                assertDone(timeLockService.startTransactionsWithWatches(startTransactionsRequestWithInitialVersion));
-
-        return response.getLockWatchUpdate().accept(LOCK_WATCH_STATE_UPDATE_VISITOR);
-    }
-
-    private static <T> T assertDone(Future<T> future) {
-        assertThat(future).isDone();
+        ListenableFuture<ConjureStartTransactionsResponse> responseFuture =
+                timeLockService.startTransactionsWithWatches(startTransactionsRequestWithInitialVersion);
+        assertThat(responseFuture).isDone();
         try {
-            return future.get();
+            return responseFuture.get().getLockWatchUpdate().accept(LOCK_WATCH_STATE_UPDATE_VISITOR);
```

Comment:
You can do `AtlasFutures::getUnchecked` so you don't need to wrap/catch runtime exceptions.